src/applications/vaos/services/appointment/transformers.js

```diff
@@ -278,6 +278,7 @@ export function transformVAOSAppointment(
     // NOTE: Timezone will be converted to the local timezone when using 'format()'.
     // So use format without the timezone information.
     start: !isRequest ? start.format('YYYY-MM-DDTHH:mm:ss') : null,
+    startUtc: !isRequest ? appt.start : null,
```

Comment:
What's the difference between appointment localStartTime and start?

Reply:
- `localStartTime` is formatted in the timezone of the appointment facility, e.g. `2025-03-27T11:00:00.000-06:00`. We actually ignore the milliseconds and offset portion of the response from vets-api but we correctly use the rest as the time to display to the user (i.e. we show the local time on appointment list and details pages).
- `start` is formatted in the UTC timezone, e.g. `2025-03-27T17:00:00Z`
Note that the examples are retrieved from Judy's account in staging.

Reply:
Currently, only the ICS calendar file needs to be in UTC so it's the only one that uses it. Everything else in the web app still uses `start` which is the local time.